### PR TITLE
Updated Tooltips on Circle Admin page

### LINKED ISSUE #428: Circle Admin Buttons Tooltip Addition
### **Summary**
After users create a circle, they are unsure how the application works and the terms Coordinape uses to describe its functions. Adding tooltips to the terms "Add Contributors" and "Create Epoch" will help users quickly gain context. 

### **Specification**
- Remove the Add circle button from the upper right-hand corner and add it to the drop-down. 
- On the Add Contributor button, remove the plus icon. Then, add the "i" icon on the Add, Contributor button. The icon should trigger a tooltip. 
- On the Add Epoch button, remove the plus icon and change the button copy to "Create Epoch"  On the button, add the "i" icon. The icon should trigger a tooltip. For example, change the button copy  "Create Epoch."

Add Contributor Tooltips Specification 
"A member of a circle that can receive GIVE or kudos for contributions performed."
Link - https://docs.coordinape.com/welcome/gift_circle#the-gift-circle

Create Epoch Tooltips Specification 
"An Epoch is a period of time where circle members contribute value & allocate GIVE tokens to one another."
Link - https://docs.coordinape.com/welcome/how_to_use_coordinape#my-epoch

[Desktop Prototype](https://www.figma.com/proto/KciBaNjP5Q9idchsxMp9ek/Coordinape-Tool-Tip-Additions?page-id=0%3A1&node-id=28%3A2967&viewport=241%2C48%2C0.71&scaling=min-zoom&starting-point-node-id=28%3A2967&show-[proto-sidebar=1&hide-ui=1) 
[Mobile Prototype](https://www.figma.com/proto/KciBaNjP5Q9idchsxMp9ek/Coordinape-Tool-Tip-Additions?page-id=0%3A1&node-id=49%3A1046&viewport=241%2C48%2C0.71&scaling=scale-down-width&starting-point-node-id=49%3A1046&show-proto-sidebar=1&hide-ui=1)

### **User Story** 
I just started a new circle but understand what circles and epochs are. 

**Acceptance Criteria** 
1  Scenario: User does not know the term Contributor
Given: The user navigates to the circle admin page
And: On a desktop device
When: The user hovers over the "i" icon next to "Add Contributor."
Then: The system triggers a tooltip
When: The user clicks to the tooltip link 
Then: The system opens a new tab to the Gift Circle Doc 

2  Scenario: User does not know the term Contributor
Given: The user navigates to the circle admin page
And: On a mobile device
When: The user presses and holds over the button "Add Contributor."
Then: The system triggers a tooltip
When: The user clicks to the tooltip link 
Then: The system opens a new tab to the My Epoch section of the support doc 

3  Scenario: User does not know the term Epoch
Given: The user navigates to the circle admin page
And: On a desktop device
When: The user hovers over the "i" icon next to "Create Epoch
Then: The system triggers a tooltip
When: The user clicks to the tooltip link 
Then: The system opens a new tab to the My Epoch section of the support doc 

4  Scenario: User wants to create a new circle
Given: The user navigates to the circle admin page
When: The user clicks their profile picture
Then: The system triggers a drop-down
When: The user clicks "Create a New Circle."
Then: The system navigates to the Create a Circle page.

5  Scenario: User wants to create a new circle
Given: The user navigates to the circle admin page
When: The user clicks the hamburger menu
Then: The system triggers a drop-down
When: The user clicks "Create a New Circle."
Then: The system navigates to the Create a Circle page.

### **Functional Requirement**
Tooltips must appear long enough for users to navigate the tooltip link for approximately 2 seconds. 

### **Assumption** 
Developers can configure tooltip interactions in seconds. For example, Front-End can easily add the material design "i" icon to buttons. 

### **Stakeholders**
Alex, the art director  &  Sascha, UI Designer

### PR DESCRIPTION
**Pull request by teeolendo**
Closes #428 by updating the Tooltip Component to accept a custom styling prop and moved the Add Circle button to the top right nav.

### CODE REVIEW

**Review by levity — CHANGES_REQUESTED** [review]
In addition to the line items, please make sure that the new info icons' size, vertical alignment, & spacing from text matches the Settings and Add Circle buttons' icons

**Review by exrhizo — COMMENTED** [review]
I agree w/ Lawrence, the info icon ought to match the add circle circle size, which might be too small...

Also noticing that as edax has started us on the path to using stitches, it might not be worth doing much to perfect the design, but I also notice that the info box fades in and out subtly in Ivy's design.

File: src/pages/AdminPage/AdminPage.tsx [hunk 1]
```diff
@@ -473,7 +501,16 @@ const AdminPage = ({ legacy }: { legacy?: boolean }) => {
                 variant="contained"
                 color="primary"
                 size="small"
-                startIcon={<PlusCircleIcon />}
+                endIcon={
+                  <ApeInfoTooltip
+                    iconTheme={classes.infoIcon}
+                    enterNextDelay={1}
+                    placement="bottom"
+                    leaveDelay={2000}
```
[the remark this answers is not in the document]
Comment (teeolendo) [reply src/pages/AdminPage/AdminPage.tsx:509]: @ivyquinzel would you be ok with 500ms instead of 2secs?

**Comment by ivyquinzel** [comment]
As long as the user has enough time to click on the link inside the tool tip

**Review by levity — APPROVED** [review]
looks good!

**Pull request merged**